packages/devextreme-schematics/src/add-layout/files/src/app/shared/components/side-navigation-menu/side-navigation-menu.component.ts

```diff
@@ -10,7 +10,7 @@ import * as events from 'devextreme/events';
   styleUrls: ['./side-navigation-menu.component.scss']
 })
 export class SideNavigationMenuComponent implements AfterViewInit, OnDestroy {
-  @ViewChild(DxTreeViewComponent<% if(requireStaticFlag) { %>, { static: true }<% } %>)
+  @ViewChild(DxTreeViewComponent, { static: true })
```

Comment:
Are you sure it will work with all supported versions of Angular?

Reply:
Yes. Angular [supports](https://angular.io/guide/releases#support-policy-and-schedule) versions 8-10

Reply:
This flag was added in 8 version